<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editcomputer</class>
 <widget class="QDialog" name="editcomputer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="table_update_computer">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Computer type</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_Name">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="input_Computer_Name"/>
   </item>
   <item>
    <widget class="QLabel" name="label_Constr">
     <property name="text">
      <string>Was it constructed?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_constr"/>
   </item>
   <item>
    <widget class="QLabel" name="label_buildy">
     <property name="text">
      <string>Build year</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="input_Computer_buildy"/>
   </item>
   <item>
    <widget class="QLabel" name="label_Model">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="input_Computer_Model"/>
   </item>
   <item>
    <widget class="QLabel" name="label_test">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_edit_computer_dialog">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
